packages/blockchain-extension/extension/commands/importNodesToEnvironmentCommand.ts

```diff
@@ -360,11 +360,11 @@ export async function importNodesToEnvironment(environmentRegistryEntry: FabricE
 
     } catch (error) {
         if (fromConnectEnvironment) {
-            throw error;
+            return error;
         }
         outputAdapter.log(LogType.ERROR, `Error ${methodMessageString}ing nodes: ${error.message}`);
         if (fromAddEnvironment) {
-            throw error;
+            return error;
```

Comment:
`ExtensionCommands.EDIT_NODE_FILTERS` and `ExtensionCommands.IMPORT_NODES_TO_ENVIRONMENT` used in addEnvironmentCommand.ts. Need to handle return error there.